src/components/basic/ImageGallery/index.tsx

```diff
@@ -52,6 +52,12 @@ export const ImageGallery = ({
     else return 3
   }
 
+  const getIsInfinite = () => {
+    if (mobile) return gallery?.length > 1
+    else if (tab) return gallery?.length > 2
+    else return gallery?.length > 3
```

Comment:
please remove `else` in both lines above because it's not needed when return in the if clause is guaranteed